pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -78,6 +78,7 @@ _period_code_map = {
     "Q-NOV": 2011,    # Quarterly - November year end
 
     "M": 3000,        # Monthly
+    "MS": 3000,       # Monthly
```

Comment:
why do we think adding this here is a good idea? this makes the map not invertible any more.

Reply:
I'll rework it, but probably in the second half of the week.

Reply:
fixed